src/pages/gateway/cicd.md

```diff
@@ -0,0 +1,237 @@
+      - name: npm install
+        run: npm i
+      - name: Setup CLI
+        uses: adobe/aio-cli-setup-action@1.3.0
+        with:
+          os: ${{ matrix.os }}
+          version: 10.x.x
+      - name: api-mesh-plugin install
+        run: aio plugins:install @adobe/aio-cli-plugin-api-mesh
+      - name: Auth
+        uses: adobe/aio-apps-action@3.3.0
+        with:
+          os: ${{ matrix.os }}
+          command: oauth_sts
+          CLIENTID: ${{ secrets.CLIENTID_PROD }}
+          CLIENTSECRET: ${{ secrets.CLIENTSECRET_PROD }}
+          TECHNICALACCOUNTID: ${{ secrets.TECHNICALACCID_PROD }}
+          TECHNICALACCOUNTEMAIL: ${{ secrets.TECHNICALACCEMAIL_PROD }}
+          IMSORGID: ${{ secrets.IMSORGID_PROD }}
+          SCOPES: AdobeID, openid, read_organizations, additional_info.projectedProductContext, additional_info.roles, adobeio_api, read_client_secret, manage_client_secrets
+      - name: Select org
+        run: aio console:org:select 123456
+      - name: Select project
+        run: aio console:project:select 1234567890
+      - name: Select workspace
+        run: aio console:workspace:select 12345678901
+      - name: 'Create env file'
+        run: |
+          touch .env
+          echo API_ENDPOINT="https://xxx.execute-api.us-west-2.amazonaws.com" >> .env
+          echo API_KEY=${{ secrets.API_KEY }} >> .env
+          cat .env
+      - name: api-mesh update
+        run: aio api-mesh:update -c meshConfig.json --env .env 
+```
+
+This script sets up the CLI, installs the API Mesh plugin (`aio-cli-plugin-api-mesh`), and adds your OAuth token. Once the workflow is configured, running its corresponding job on the GitHub **Actions** tab will update your mesh.
+
+Uncomment the following section to automatically trigger this workflow when a PR is merged `main`:
+
+```yml
+push:
+  branches: [ "main" ]
+```
+
+Uncomment the following section to automatically trigger this workflow when a PR to `main` is updated:
+
+```yml
+pull_request:
+  branches: [ "main" ]
+```
+
+For more information see [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).
+
+## Secrets and variables
+
+When developing locally, you can store your variables and secrets in your [environment variables file](./developer-tools.md#environment-variables). When using GitHub, you will need to use [secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) and [variables](https://docs.github.com/en/actions/learn-github-actions/variables).
+
+- Variables are for information that is not sensitive, for example, if you are targeting `stage` or `prod`. When viewing your mesh in the Developer Console or by running the `aio api-mesh:get` command, these variables will be exposed.
+
+- Secrets are for sensitive information, such as access tokens. These are never exposed in the Developer Console or when running the `aio api-mesh:get` command.
```

Comment:
@revanth0212 is this statement correct in current version?

Reply:
No @naomirobichaud I believe the wording needs to be adjusted to better match our current offering.

There could be confusion here because we are talking about Github Variables and Github Secrets (not API Mesh) and unfortunately we will be using the same naming convention for our future feature **Secrets Management**.

This statement will be valid once we launch our feature later this year.

Meanwhile, I believe we can adjust the wording to only talk about Github Secrets.

```suggestion
- Secrets are for sensitive information, such as access tokens. These are never exposed in the Github code repo.
```